lexer.js

```diff
@@ -487,6 +529,34 @@ function tryParseObjectDefineOrKeys (keys) {
           if (ch !== quot) break;
           pos += 1;
           ch = commentWhitespace();
+          if (ch === 38/*&*/) {
+            if (source.charCodeAt(pos + 1) !== 38/*&*/) break;
+            pos += 2;
+            ch = commentWhitespace();
+            if (ch !== 33/*!*/) break;
+            pos += 1;
+            ch = commentWhitespace();
+            if (source.startsWith(id, pos)) {
```

Comment:
Which tool uses `exports.hasOwnProperty`?

Reply:
RollupJS just does this now as per #38. I'm not sure if there's anything for the collision case, that's why I added the `Object.hasOwnProperty` as well in case that gets added in future //cc @lukastaegert.